test_phage_reference/3_validate_expression_data.ipynb

```diff
@@ -558,183 +1048,133 @@
-       "      <td>0.0</td>\n",
-       "      <td>0.0</td>\n",
-       "      <td>0.0</td>\n",
-       "      <td>0.0</td>\n",
-       "      <td>0.0</td>\n",
-       "      <td>...</td>\n",
-       "      <td>0.0</td>\n",
-       "      <td>0.0</td>\n",
-       "      <td>0.0</td>\n",
-       "      <td>0.0</td>\n",
-       "      <td>0.0</td>\n",
-       "      <td>0.0</td>\n",
-       "      <td>0.0</td>\n",
-       "      <td>0.0</td>\n",
-       "      <td>0.0</td>\n",
-       "      <td>0.0</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n",
-       "<p>2 rows × 1128 columns</p>\n",
-       "</div>"
-      ],
       "text/plain": [
-       "             NC_028999.1  MT133560.1  MK599315.1  MH725810.1  MF974178.1  \\\n",
-       "SRR13196071          0.0         0.0         0.0         0.0         0.0   \n",
-       "SRR13196068          0.0         0.0         0.0         0.0         0.0   \n",
-       "\n",
-       "             NC_016765.1  NC_031063.1  NC_027375.1  NC_011810.1  MT108726.1  \\\n",
-       "SRR13196071          0.0          0.0          0.0          0.0         0.0   \n",
-       "SRR13196068          0.0          0.0          0.0          0.0         0.0   \n",
-       "\n",
-       "             ...  DI373497.1  DI373496.1  DI373495.1  DI373494.1  DI373493.1  \\\n",
-       "SRR13196071  ...         0.0         0.0         0.0         0.0         0.0   \n",
-       "SRR13196068  ...         0.0         0.0         0.0         0.0         0.0   \n",
-       "\n",
-       "             DI373492.1  DI373491.1  DI373490.1  DI373489.1  DI373488.1  \n",
-       "SRR13196071         0.0         0.0         0.0         0.0         0.0  \n",
-       "SRR13196068         0.0         0.0         0.0         0.0         0.0  \n",
-       "\n",
-       "[2 rows x 1128 columns]"
+       "971961.851541"
       ]
      },
-     "execution_count": 10,
+     "execution_count": 13,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "expression_phage_df.loc[[\"SRR13196071\", \"SRR13196068\"]]"
+    "# Check there is expression in PAO1 sample using phage reference gene: NC_010326.1\n",
+    "expression_phage_df.loc[\"SRR13160334\", \"NC_010326.1\"]"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 18,
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Takeaway:**\n",
+    "\n",
+    "* Most genes in PAO1 sample are nonzero, including phage gene that the sample is known to contain -- as expected\n",
+    "* Is the TPM < 1000 expected?\n",
+    "* Why is TPM of phage gene higher than other PAO1 genes?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
    "metadata": {},
+   "source": [
+    "## Positive case: PA14\n",
+    "\n",
+    "* Check that PA14 sample (ERR3642743) aligns well to PA14 and phage references. \n",
+    "* Check that PA14 sample has expression at Pf5 (PA14_49010, PA14_49020 in PA14 genome) \n",
```

Comment:
Just noting that there isn't enough context here for me to understand this assumption

Reply:
Very good point. I will add more description here so that it's clear